Dockerfiles/midstream/run_tests.py

```diff
@@ -42,24 +42,45 @@ def run_validate_operator_bundle(self):
                     -e 'work_dir={work_dir}'".format(operator_dir=self.operator_dir,
                                                     operator_work_dir=self.operator_work_dir,
                                                     work_dir=self.work_dir)
-        result = subprocess.run(exec_cmd, shell=True)
+        result = subprocess.run(exec_cmd, shell=True, capture_output=True)
         return result
 
     def test_for_extract_and_validate_bundle_image(self):
+        
+        self.setUp()
+        global exit_code
+        exit_code = 0
         result = self.run_extract_operator_bundle()
-        self.assertEqual(0,
-                         result.returncode,
-                         "extract-operator-bundle.yml failed")
+        if(result.returncode != 0):
+            print("Ansible playbook extract-operator-bundle.yml failed with result code: %s , see the file extract-operator-bundle.errormessage for more info." % result.returncode)
```

Comment:
The requirements in the story says:
```
.errormessage 
Should be place in $PWD (where the entrypoint script is located).
```
The requirements dows not say anything about `extract-operator-bundle.errormessage`

Reply:
Changed to .errormessage

Reply:
@jsztuka I still cannot see the change.